indigo/distribution.yaml

```diff
@@ -4580,6 +4580,45 @@ repositories:
       url: https://github.com/GT-RAIL/jaco_gazebo.git
       version: develop
     status: maintained
+  jaguar:
+    doc:
+      type: git
+      url: https://github.com/gstavrinos/jaguar.git
+      version: kinetic-devel
+    release:
+      packages:
+      - jaguar_control
+      - jaguar_description
+      - jaguar_msgs
+      - jaguar_navigation
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
Thanks for pointing that out. The wiki was not clear on this. The stack has been tested on all three distributions, so I will just have to run each distribution in a different Ubuntu version, just to be sure. Thanks again.